bootstrap/src/uri/mvd-uri.ts

```diff
@@ -116,10 +116,14 @@ export class MvdUri implements ZLUX.UriBroker {
   }
 
   pluginResourceUri(pluginDefinition: ZLUX.Plugin, relativePath: string): string {
+    let web = 'web/';
     if (relativePath == null) {
       relativePath = "";
     }
-    return `${this.pluginRootUri(pluginDefinition)}web/${relativePath}`;
+    if(relativePath==='iframe') {
+      web = ''
+    }
+    return `${this.pluginRootUri(pluginDefinition)}${web}${relativePath}`;
```

Comment:
previously, if someone were to do 
`pluginResourceUri(pluginDef, 'iframe')` they would have received
....../web/iframe
Now they would recieve
....../iframe
And, in order to get what they used to have, it would be
`pluginResourceUri(pluginDef, 'web/iframe')`

Realistically I don't think anyone was doing that, but it makes me think this is not a good design for the function.
Why not make a new function, `pluginIframeUri`? Separate responsibilities. All it requires is to additionally add an entry here https://github.com/zowe/zlux-platform/blob/staging/interface/src/index.d.ts#L229 and to make the same change in `bootstrap/src/uri/dsm-uri.ts`, which is historical but will break the build if you dont.

Reply:
ok sure, can add a new function

Reply:
refactored with PR https://github.com/zowe/zlux-platform/pull/68/files